### ISSUE TITLE
Inconsistent naming of DataSpecificationIec61360

### ISSUE THREAD

**Issue by arnoweiss** [admin-shell-io/aas-specs#263]
Currently, the V3-schemas for [JSON](https://github.com/admin-shell-io/aas-specs/blob/6f7d5b1e386d950cbd02c477d80065fb2e95fa99/schemas/json/aas.json#L363) and [XML](https://github.com/admin-shell-io/aas-specs/blob/6f7d5b1e386d950cbd02c477d80065fb2e95fa99/schemas/xml/AAS.xsd#L212) use the spelling `DataSpecificationIec61360`. However, the schemas in [RDF and SHACL](https://github.com/admin-shell-io/aas-specs/tree/master/schemas/rdf) use the spelling [`DataSpecificationIEC61360`](https://github.com/admin-shell-io/aas-specs/blob/6f7d5b1e386d950cbd02c477d80065fb2e95fa99/schemas/rdf/rdf-ontology.ttl#L438). This difference is not only inconsistent and therefore impractical in itself. It ripples on and breaks the AAS4J serializers. 

Therefore, I suggest to adjust the naming to `DataSpecificationIec61360` in RDF and SHACL as well.

**Comment by mjacoby** [comment]
This also affects the following elements
- `DataTypeIEC61360`
- `LangStringDefinitionTypeIEC61360`
- `LangStringPreferredNameTypeIEC61360`
- `LangStringShortNameTypeIEC61360`

**Comment by BirgitBoss** [comment]
Yes, this seems to be inconsistent and should be fixed. @sebbader-sap : we then also need a new release

**Comment by sebbader-sap** [comment]
Correct, will take a look together with @s-heppner
